message ScaleScore {
    int32 scale_id = 1;
    float score = 2;
}

message CareerMatch {
    int32 career_id = 1;
    float score = 2;
}

message CareerMatchesRequest {
    repeated ScaleScore scale_scores = 1;
}

message CareerMatchesResponse {
    repeated CareerMatch matches = 1;
}

service CareerMatchesService {
    rpc GetMatches (CareerMatchesRequest) returns (CareerMatchesResponse);
}
